pageserver/src/tenant.rs

```diff
@@ -1823,7 +1822,10 @@ impl Tenant {
             OpenOptions::new()
                 .truncate(true) // This needed for overwriting with small config files
                 .write(true)
-                .create_new(first_save),
+                .create_new(first_save)
+                // when initializing a new tenant, first_save will be true, thus this flag will be
+                // ignored (per rust std docs). later, don't care.
```

Comment:
NIT: The comment is somewhat confusing: first, we ignore the parameter, then we don't care? What's the difference? 🙂

Reply:
+1 for confusion. I think it refers to combination of `.create_new` + `.create` and the way its handled on stdlib side

Reply:
> I think it refers to combination of `.create_new` + `.create`

Yes, correct. I'll try to word this better.

Reply:
New comment pushed.

Reply:
Approvals so I assume both are happy with the de-obfuscation. Apologies for needing the extra roundtrip for these, I cannot seem to shake this habit off.

Reply:
Thats better to have an extra roundtrip to ensure everyone is on the same page, so thats on me (I forgot  to resolve the discussion :))